src/demand.lua

```diff
@@ -119,53 +122,72 @@ function demand.getDemandFactorInfo()
 	return utils.deepCopy(DemandFactor)
 end
 
---- Updates the demand factor and returns the updated demand factor
+--- Gets the period for the timestamp
 --- @param timestamp number The current timestamp
---- @return number | nil # The demand factor, updated if necessary, nil if no update is necessary
-function demand.updateDemandFactor(timestamp)
-	if not demand.shouldUpdateDemandFactor(timestamp) then
-		return nil
-	end
-
-	local settings = demand.getSettings()
-
-	-- check that we have settings
-	if not settings then
-		return nil
-	end
+--- @return number # The period for the timestamp
```

Comment:
Might want to note whether this is 1 or 0 indexed

Reply:
[e162b6f](https://github.com/ar-io/ar-io-network-process/pull/379/commits/e162b6f78b2652c4c398e4ef3a57f9168a45352e)